modules/client-configuration/pages/repositories.adoc

```diff
@@ -16,7 +16,13 @@ For more information about custom repositories and channels, see xref:administra
 
 You can configure local repositories on Salt clients, to provide packages that are not supplied by {productname} channels.
 
-Local repositories are disabled during onboarding.
+[WARNING]
+====
+In general, client systems should have local repositories configured.
+Local repositories can cause management because you otfen do not know what is available on the client system and you can get packages installed you did not expect.
```

Comment:
```suggestion
Local repositories can cause management troubles because you do not know what is available on the client system and you can get packages installed you did not expect.
```